pkg/networkservice/chains/nsmgr/server.go

```diff
@@ -112,24 +122,16 @@ func NewServer(ctx context.Context, nsmRegistration *registryapi.NetworkServiceE
 		localbypass.NewServer(&localbypassRegistryServer),
 		excludedprefixes.NewServer(ctx),
 		newRecvFD(), // Receive any files passed
-		interpose.NewServer(nsmRegistration.Name, &interposeRegistry),
+		interpose.NewServer(nsmRegistration.Name, &interposeRegistry, connectServer),
```

Comment:
@denis-tingajkin We already discussed before, and I think we should simply had a context variable describing we are going into cross connect NSE, and just modify filter mechanism to check for it. Instead of making a chain jumpings.

Reply:
@haiodo I think context variable is a more complex solution. Because in this case all elements after interpose should know about two interpose branches...